app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -1,23 +1,26 @@
 class RecurringContribution::Subscriptions::Processor
-  attr_reader :subscription
+  attr_reader :subscription, :with_save
 
-  def initialize(subscription, credit_card_hash)
+  def initialize(subscription, credit_card_hash, with_save)
```

Comment:
Wouldn't it be better to have a named parameter here? If you think about the call site of this constructor you'd see `RecurringContribution::Subscriptions::Processor.new(subscription, { card_code: '123123123' } , true)`, the person who see this code will have no clue about what the `true` is.